# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../unclutter/fix-makefile.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- unclutter/Makefile.vanilla	2008-12-19 22:59:30.000000000 +0100
+++ unclutter/Makefile	2008-12-19 23:08:24.000000000 +0100
@@ -23,11 +23,11 @@
         IMAKE_CMD = $(IMAKE) -DUseInstalled -I$(IRULESRC) $(IMAKE_DEFINES)
            BINDIR = $(DESTDIR)/usr/bin/X11
           INCROOT = $(DESTDIR)/usr/include
-          MANPATH = $(DESTDIR)/usr/catman/x11_man
+          MANPATH = $(DESTDIR)/usr/X11/man
     MANSOURCEPATH = $(MANPATH)/man
            MANDIR = $(MANSOURCEPATH)1
             IMAKE = imake
-             XLIB = $(EXTENSIONLIB)  -lX11
+             XLIB = $(EXTENSIONLIB) $(shell pkg-config x11 --libs)
 
   LOCAL_LIBRARIES = $(XLIB)
 
@@ -44,7 +44,6 @@
 
 install:: unclutter
 	$(INSTALL) -c $(INSTPGMFLAGS)   unclutter $(BINDIR)
-install.man:: unclutter.man
 	$(INSTALL) -c $(INSTMANFLAGS) unclutter.man $(MANDIR)/unclutter.1
 clean::
 	$(RM) $(PROGRAM)
